web-client/src/ducks/requests/reducer.ts

```diff
@@ -1,17 +1,22 @@
-import { Request } from 'src/models/requests';
+// import TimelineItem from 'antd/lib/timeline/TimelineItem';
```

Comment:
Change Offer to My Cav Request and Request to My Pin and create separate variables for FIND.